### PR TITLE
[TASK] Revise installation chapter

### PR DESCRIPTION
**Pull request by brotkrueml**
- Add a TOC
- Streamline wordings
- Add some references where appropriate

Releases: main, 12.4

### CODE REVIEW

File: Documentation/Installation/Install.rst [hunk 1]
```diff
@@ -126,122 +133,132 @@ Interactive / guided setup (questions/answers):
...
-Initialise
------------
+Initialize
+----------
 
-TYPO3 offers two options for initialisation: creating an empty starting page or
-it can go directly to the backend administrative interface.
-Beginners should select the first option and allow TYPO3 to create an empty starting page.
+TYPO3 offers two options for initialization: creating an empty starting page or
+it can go directly to the backend administrative interface. Beginners should
```
Comment (linawolf) [inline Documentation/Installation/Install.rst:248]: Not sure if this is always the case. If you plan to start with the introduction package for example it is better not to choose an empty starting page?

**Review by linawolf — COMMENTED** [review]
Great work! Just some small comments

**Comment by linawolf** [comment]
@sarahmccarthy123 I would be happy if you could also have a look at this. I think this chapter is very crucial for beginners

File: Documentation/Installation/Install.rst [hunk 2]
```diff
@@ -55,23 +61,24 @@ At the root level of your web server, execute the following command:
             cd example-project-directory
 
             # Tell DDEV to create a new project of type "typo3"
-            # 'docroot' MUST be 'public' PHP 8.1 is required by TYPO3 v12
+            # 'docroot' MUST be set to 'public'
+            # At least PHP 8.1 is required by TYPO3 v12. Adapt the PHP version to your needs.
             ddev config --project-type=typo3 --docroot=public --create-docroot --php-version 8.1
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/Installation/Install.rst:66]: One of them, yes, I assume. But the user should choose the version according to the live server, so I would recommend this with the comment to adjust it when necessary. This is then explicitly chosen by the user.

File: Documentation/Installation/Install.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/Installation/Install.rst:248]: Don't know. haven't used the introduction package yet. This PR is just a "I make it a little better" not a complete overhaul with checking everything written and testing all the variants. This should be done at some point in time, but currently not having time for that.

File: Documentation/Installation/Install.rst [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
-===============================
 
-The following script will install TYPO3 v12 which is the latest release of the CMS. If you wish to
-install the long term support (LTS) release of TYPO3, visit the :ref:`TYPO3 v11
-Installation guide <t3start11:install>`.
+Create the project with Composer
+================================
+
+The following command will install TYPO3 v12. If you want to install another
+version of TYPO3 use the version selector on the left side.
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/Installation/Install.rst:35]: Could we offer some version switch buttons here As we have them in othere places in Getting started?

File: Documentation/Installation/Install.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Installation/Install.rst:248]: Maybe add a todo or issue to check that later?

Comment (brotkrueml) [reply Documentation/Installation/Install.rst:248]: As far as I remember, you have only these two options: Create an empty start page or create nothing. I assume, the introduction package is then an extension that has to be added later.

**Review by linawolf — APPROVED** [review]

**Pull request merged**